_includes/code-snippet.html

```diff
@@ -1,7 +1,7 @@
 <div class="code-snippet-area">
   {% unless include.nocopy %}
   <div class="code-snippet-buttons">
-    <button class="copy-button"><i class="fa fa-clone"></i></button>
+    <button class="code-snippet-button copy-button"><i class="far fa-clone"></i></button>
```

Comment:
Is “far” intended?

Reply:
yeah because with the new font-awesome it inverts the symbol without this change